joystreamjs/content/tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
+  "compilerOptions": {
+    "declaration": true,
+    "importHelpers": true,
+    "module": "commonjs",
```

Comment:
I tried converting `@joystream/js` to ES module, but unfortunately it didn't work quite well with other monorepo packages whom its a dependency, e.g. CLI. Since, `@joystream/cli` is a commonJS module while packing it I was getting weird non-existing module resolution errors

Will have to look more into this otherwise, I will try to make this package hybrid

Reply:
@zeeshanakram3 any update on this issue? If doing hybrid is super hard, we can probably live with CJS but it would be much easier for us to have ESM available

Reply:
> we can probably live with CJS but it would be much easier for us to have ESM available

 Yeah you are right about the benefit, but I think the cost is that after conversion to ESM, the packages(`cli`/`network-tests`) that use `@joystream/js` as dependency will have to change their importing/module-resolution behaviour.
 Anyways, I did some preliminary work but didn't fully pursue it. So please let me if this is absolutely required, I will start working on it.

Reply:
Addressed in [9158a05](https://github.com/Joystream/joystream/pull/4334/commits/9158a05f44af39fa371da9052a5579c0ad18d880)